### PR TITLE
Migration data

### LINKED ISSUE #55: Migration data
Data for the migration indicator needs to be retrieved.

At this point I found only this dataset:
https://opendata.cbs.nl/statline/portal.html?_catalog=CBS&_la=nl&tableId=37713&_theme=265

It shows the migration background, which can be used but I was looking for something that includes migration within the municipalities as well.

### PR DESCRIPTION
**Pull request by Augustin4762**

### CODE REVIEW

**Comment by Augustin4762** [comment]
Oh, the conflict with the get_data file is due to the fact that the branch migration was created before the recent update of the get_data function. If you could please make the necessary adjustments into this function.

**Review by ikbentimkramer — CHANGES_REQUESTED** [review]
Please run `devtools::check()` and resolve all errors, warnings and notes that it brings up. Will review again when that has been done.

File: tests/testthat/test-map_migration.R [hunk 1]
```diff
@@ -0,0 +1,44 @@
+test_map_migration <- function(df) {
```
Comment (ikbentimkramer) [inline tests/testthat/test-map_migration.R]: This is not a test_that function, neither does it run some kind of test. Why is this in the tests folder and called test-map_migration?

**Comment by Augustin4762** [comment]
Could you please check why is there a conflict with the get_data function? I think I had this issue before the break and couldn't figure it out.

**Review by ikbentimkramer — APPROVED** [review]

**Pull request merged**